### PR TITLE
Add Redpanda / Upstash / Kinesis to logos

### PR DESCRIPTION
**Pull request by pjhampton**
We are making UI changes internally on the control-plane for ClickPipes. This PR adds 2 Kafka protocol provider logos + AWS Kinesis to the UI lib. 

Screenshot 2024-01-09 at 10 48 07

---

### Testing

```
$ npm run storybook
```

Navigate to: Icon > Icons > Logos

### CODE REVIEW

File: src/components/Logos/Upstash.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <path d="M256,42.415426 C199.445737,-14.1384753 107.753322,-14.1384753 51.1994207,42.415426 C-5.35485714,98.9696894 -5.35485714,190.662104 51.1994207,247.216367 L76.7989048,221.616521 C34.3841124,179.200643 34.3841124,110.431151 76.7989048,68.0159962 C119.214783,25.6001177 187.984275,25.6001177 230.39943,68.0159962 L256,42.415426 Z" fill="#00C98D"></path>
+        <path d="M204.800308,93.6158424 C176.523538,65.3390727 130.676245,65.3390727 102.399475,93.6158424 C74.1219813,121.893336 74.1219813,167.739181 102.399475,196.015951 L127.999321,170.416105 C113.860936,156.27772 113.860936,133.354797 127.999321,119.215688 C142.137706,105.077304 165.060629,105.077304 179.199738,119.215688 L204.800308,93.6158424 Z" fill="#00C98D"></path>
+        <path d="M256,42.415426 C199.445737,-14.1384753 107.753322,-14.1384753 51.1994207,42.415426 C-5.35485714,98.9696894 -5.35485714,190.662104 51.1994207,247.216367 L76.7989048,221.616521 C34.3841124,179.200643 34.3841124,110.431151 76.7989048,68.0159962 C119.214783,25.6001177 187.984275,25.6001177 230.39943,68.0159962 L256,42.415426 Z" fill-opacity="0.4" fill="#FFFFFF"></path>
```
Comment (vineethasok) [inline src/components/Logos/Upstash.tsx:16]: Need to change fill-opacity to fillOpacity

**Review by vineethasok — CHANGES_REQUESTED** [review]

**Review by serdec — APPROVED** [review]

File: src/components/Logos/Redpanda.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { SVGAttributes } from "react";
+
+const Redpanda = (props: SVGAttributes<SVGElement>) => (
+  <svg
+    width="161"
```
Comment (gjones) [inline src/components/Logos/Redpanda.tsx:5]: This should be 64px in width an height, the view box should match.

File: src/components/Logos/Upstash.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { SVGAttributes } from "react";
+
+const Upstash = (props: SVGAttributes<SVGElement>) => (
+  <svg
+    width="256"
+    height="341"
+    viewBox="0 0 256 341"
```
Comment (gjones) [inline src/components/Logos/Upstash.tsx:7]: Same here

**Comment by gjones** [comment]
Hiya @pjhampton, thanks for the PR! 🎉  All looks good except the svgs themselves. I've attached correct sized versions below. Hope that helps!

[Logos.zip](https://github.com/ClickHouse/click-ui/files/13868044/Logos.zip)

**Comment by pjhampton** [comment]
Thanks, everyone! I also added AWS Kinesis to save us some time in the near future.

**Review by vineethasok — APPROVED** [review]

**Pull request merged**